jenkins/pipelines/cd/tidb-operator.groovy

```diff
@@ -0,0 +1,409 @@
+package cd
+
+String GitHash
+String ReleaseTag
+def EnableE2E = false
+
+
+final CHART_ITEMS = 'tidb-operator tidb-cluster tidb-backup tidb-drainer tidb-lightning tikv-importer'
+final TOOLS_BUILD_DIR = 'output/tkctl'
+final CHARTS_BUILD_DIR = 'output/chart'
+
+final dindYaml = '''
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dind
+spec:
+  containers:
+  - name: builder
+    image: hub.pingcap.net/jenkins/docker-builder
+    args: ["sleep", "infinity"]
+    env:
+    - name: DOCKER_HOST
+      value: tcp://localhost:2375
+  - name: dind
+    image: docker:dind
+    args: ["--registry-mirror=https://registry-mirror.pingcap.net"]
+    env:
+    - name: REGISTRY
+      value: hub.pingcap.net
+    - name: DOCKER_TLS_CERTDIR
+      value: ""
+    - name: DOCKER_HOST
+      value: tcp://localhost:2375
+    securityContext:
+      privileged: true
+    tty: true
+    readinessProbe:
+      exec:
+        command: ["docker", "info"]
+      initialDelaySeconds: 10
+      failureThreshold: 6
+'''
+final goBuildYaml = '''
+apiVersion: v1
+kind: Pod
+metadata:
+  name: golang
```

Comment:
this filed is needless

Reply:
they will be move to a separate yaml file

Reply:
removed